src/openforms/formio/rendering/tests/test_list_components.py

```diff
@@ -0,0 +1,160 @@
+from unittest.mock import patch
+
+from django.test import TestCase
+
+from openforms.submissions.rendering import Renderer, RenderModes
+from openforms.submissions.tests.factories import SubmissionFactory
+
+from ..default import ChoicesNode, SelectNode, TextNode
+from ..registry import Registry
+
+
+class FormNodeTests(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
```

Comment:
Preferably set up the test data inside the test itself rather than creating context switches. You only have a single test, so this test data is only relevant for that single test anyway.

Reply:
Okay check, i'll keep that in mind.. 👍